docs/reference/run.md

```diff
@@ -667,6 +667,17 @@ It also causes any seccomp filters to be applied later, after privileges have be
 which may mean you can have a more restrictive set of filters.
 For more details, see the [kernel documentation](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt).
 
+## Specifying an init process
+
+You can use the `--init` or `--init-path` flags to specify a process which
+should be used as the PID 1 in the container, and which binary to use for this
+init process, respectively. Specifying an init process ensures the usual
+responsibilties of an init system, such as reaping zombie processes, are
+performed inside the created container.
+
+The default init process used when specifying `--init` without `--init-path` is
+[tini](https://github.com/krallin/tini).
```

Comment:
> Why not use `exec.LookPath` then? :)

On those distros, `docker-init`, `docker-containerd`, etc, are deliberately not in `$PATH`, so that they don't auto-complete on the commandline (and are only started through the systemd unit file, using the full path)

Maybe we should leave it out for now, until the exact use-case for setting a different `--init` _per container_ is clear